templates/java/build-alt.gradle.mustache

```diff
@@ -62,20 +83,20 @@ signing {
 
 uploadArchives.repositories.mavenDeployer {
   beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-  String stagingUrl
-    if (rootProject.hasProperty('repositoryId')) {
-      stagingUrl = 'https://oss.sonatype.org/service/local/staging/deployByRepositoryId/' +
-          rootProject.repositoryId
+  String mavenRepoUrl
+    if (rootProject.hasProperty('mavenRepoUrl')) {
+      mavenRepoUrl = rootProject.mavenRepoUrl
     } else {
-      stagingUrl = 'https://oss.sonatype.org/service/local/staging/deploy/maven2/'
+      throw new GradleException("Root project must have mavenRepoUrl")
     }
   def configureAuth = {
-    if (rootProject.hasProperty('ossrhUsername') && rootProject.hasProperty('ossrhPassword')) {
-      authentication(userName: rootProject.ossrhUsername, password: rootProject.ossrhPassword)
+    if (rootProject.hasProperty('mavenUsername') && rootProject.hasProperty('mavenPassword')) {
```

Comment:
previously we were only publishing to a an oss repository, so we needed OSSRH (OSS Repository Hosting) authorization information. Since now we can publish to any URL, I used the more generic terms for maven deployment.